main.py

```diff
@@ -69,6 +69,55 @@ def get(self, namespace, start=None, end=None, method=None,
 
         return jsonify({"response": result}), 200
 
+    @listen_to('kytos.kronos.save')
+    def event_save(self, event):
+        """Save the data in one of the backends"""
+        try:
+           self.backend.save(event.content['namespace'],
+                             event.content['value'],
+                             event.content['timestamp'])
+        except KeyError:
+            result = None
+            error = None
```

Comment:
`error = None` is effectively silencing all Key Errors. It's better to catch any exception and set error to the error tuple (exception class, exception object). This way the callback can decide what to do.